test/integration/compile-api-blueprint-test.coffee

```diff
@@ -283,4 +282,26 @@ describe('compile() · API Blueprint', ->
       assert.equal(compilationResult.transactions[0].request.uri, '/honey?beekeeper=Pavan')
     )
   )
+
+  describe('with response without explicit status code', ->
+    compilationResult = undefined
+
+    beforeEach((done) ->
```

Comment:
Do we need to compile given description document before each test? Parser would not produce different results for each call, would it?

Reply:
I'll address this in a subsequent PR - I'll change it everywhere in these tests.